themes/default/content/docs/clouds/aws/get-started/deploy-stack-b.md

```diff
@@ -0,0 +1,162 @@
+---
+title_tag: Deploy the Stack | AWS
+title: Deploy stack
+h1: "Pulumi & AWS: Deploy stack"
+meta_desc: Learn how to deploy your stack to an AWS project in this guide.
+block_external_search_index: true
+---
+
+### Configure Pulumi to access your AWS account
+
+Pulumi requires cloud credentials to manage and provision resources. You must use an IAM user account that has **Programmatic access** with rights to deploy and manage resources handled through Pulumi.
+
+If you have previously <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html" target="_blank">installed</a> and <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html" target="_blank">configured</a> the AWS CLI, Pulumi will respect and use your configuration settings.
+
+If you do not have the AWS CLI installed or plan on using Pulumi from within a CI/CD pipeline, <a href="https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html#access-keys-and-secret-access-keys" target="_blank">retrieve your access key ID and secret access key</a> and then set the `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` environment variables on your workstation.
+
+{{< chooser os "linux,macos,windows" >}}
+{{% choosable os linux %}}
+
+```bash
+$ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+$ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+```
+
+{{% /choosable %}}
+
+{{% choosable os macos %}}
+
+```bash
+$ export AWS_ACCESS_KEY_ID=<YOUR_ACCESS_KEY_ID>
+$ export AWS_SECRET_ACCESS_KEY=<YOUR_SECRET_ACCESS_KEY>
+```
+
+{{% /choosable %}}
+
+{{% choosable os windows %}}
+
+```powershell
+> $env:AWS_ACCESS_KEY_ID = "<YOUR_ACCESS_KEY_ID>"
+> $env:AWS_SECRET_ACCESS_KEY = "<YOUR_SECRET_ACCESS_KEY>"
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
+For additional information on setting and using AWS credentials, see [AWS Setup](/registry/packages/aws/installation-configuration/).
+
+### Deploy stack
```

Comment:
I think from this point on, all of the steps are identical. Do we still need the `-b` pages then? Or does having them with their `-b` URLs just make the test easier?

Reply:
It makes the navigation work. Otherwise folks on the "B" version will start seeing the "Control" if they use the Previous Step button. There is some logic in the partial I didn't want to touch as part of the test since I would have to unwind it at the end anyway. The `-b` pages just make it easier to clean up or promote forward when the test is done.